src/main/java/io/rebloom/client/cms/CMS.java

```diff
@@ -0,0 +1,96 @@
+package io.rebloom.client.cms;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Interface for RedisBloom Count-Min Sketch Commands
+ * 
+ * @see <a href=
+ *      "https://oss.redislabs.com/redisbloom/CountMinSketch_Commands/">RedisBloom
+ *      Count-Min Sketch Documentation</a>
+ */
+public interface CMS {
+	/**
+	 * CMS.INITBYDIM Initializes a Count-Min Sketch to dimensions specified by user.
+	 * 
+	 * @param key   The name of the sketch
+	 * @param width Number of counter in each array. Reduces the error size
+	 * @param depth Number of counter-arrays. Reduces the probability for an error
+	 *              of a certain size (percentage of total count
+	 */
+	void cmsInitByDim(String key, Long width, Long depth);
+
+	/**
+	 * CMS.INITBYPROB Initializes a Count-Min Sketch to accommodate requested
+	 * capacity.
+	 * 
+	 * @param key         The name of the sketch.
+	 * @param error       Estimate size of error. The error is a percent of total
+	 *                    counted items. This effects the width of the sketch.
+	 * @param probability The desired probability for inflated count. This should be
+	 *                    a decimal value between 0 and 1. This effects the depth of
+	 *                    the sketch. For example, for a desired false positive rate
+	 *                    of 0.1% (1 in 1000), error_rate should be set to 0.001.
+	 *                    The closer this number is to zero, the greater the memory
+	 *                    consumption per item and the more CPU usage per operation.
+	 */
+	void cmsInitByProb(String key, Double error, Double probability);
+
+	/**
+	 * CMS.INCRBY Increases the count of item by increment
+	 * 
+	 * @param key       The name of the sketch
+	 * @param item      The item which counter to be increased
+	 * @param increment Counter to be increased by this integer
+	 * @return Count for the item after increment
+	 */
+	Long cmsIncrBy(String key, String item, Long increment);
+
+	/**
+	 * CMS.INCRBY Increases the count of one or more item.
+	 * 
+	 * @param key            The name of the sketch
+	 * @param itemIncrements a Map of the items to be increased and their integer
+	 *                       increment
+	 * @return Count of each item after increment
+	 */
+	List<Long> cmsIncrBy(String key, Map<String, Long> itemIncrements);
```

Comment:
The indexes of returned `List<Long>` (probably) have direct relationship with the entries of `Map<String, Long>`. In such case `Map.Entry<String, Long>...` would be a better choice here.

Reply:
Yeah, I didn't want to change the API too radically from the underlying Redis command which returns a List/Array. I looked at our other drivers and they don't seem to change the datatype/structure of the return, so I keep it as a List. See https://oss.redislabs.com/redisbloom/CountMinSketch_Commands/#cmsincrby
My feeling is that these drivers are pretty "close-to-the-metal" and they should have one API that is 1-to-1 to the underlying Redis Commands, but I agree that we can/could/should add a higher-level of abstraction methods perhaps in a separate package. Ideas?